apps/condo/domains/ticket/hooks/useCacheUtils.tsx

```diff
@@ -0,0 +1,35 @@
+import { ApolloCache } from '@apollo/client/cache'
+import { generateQueryVariables } from '@condo/domains/common/components/TicketCard/TicketCardList'
+import { Ticket as TicketGQL } from '@condo/domains/ticket/gql'
+import { ITicketUIState } from '@condo/domains/ticket/utils/clientSchema/Ticket'
+
+interface ICachedData {
+    objs: ITicketUIState[],
+}
+
+type TicketCacheUtilsHook = (cache: ApolloCache<unknown>) => {
+    addTicketToQueryCacheForTicketCardList: (ticket: ITicketUIState) => void
+}
+
+export const useCacheUtils: TicketCacheUtilsHook = (cache) => {
+    function addTicketToQueryCacheForTicketCardList (ticket) {
+        const queryData = {
+            query: TicketGQL.GET_ALL_OBJS_WITH_COUNT_QUERY,
+            variables: generateQueryVariables(ticket.organization.id, ticket.clientPhone),
+        }
+
+        const cachedData: ICachedData = cache.readQuery(queryData)
+
+        if (cachedData) {
+            cache.writeQuery({
+                ...queryData,
+                data: {
+                    ...cachedData,
+                    objs: [...cachedData.objs, ticket],
```

Comment:
It seems, like you should insert new record at the beginning of array to respect a sort order `TICKET_SORT_BY = ['createdAt_DESC']` used in `generateQueryVariables`. Check this, please.

Reply:
It's not necessary, because because data sorted by apollo. I've tested, all is ok.